templates/signed-out.html

```diff
@@ -1,11 +1,34 @@
-{% extends 'layouts/_base.html' %}
+{% extends 'errors/_base.html' %}
+
+{% from "components/panel/_macro.njk" import onsPanel %}
 
 {% set page_title = _("Signed out") %}
 
 {% block main %}
-  <h1>{{ _("Your survey answers have been saved. You are now signed out") }}.</h1>
 
-  {% if account_service_log_out_url %}
-    <a href="{{account_service_log_out_url}}">{{ _("Return to your account") }}</a>
+  {{
+    onsPanel({
+      "id": '',
+      "type": 'success',
+      "iconType": 'check',
+      "iconSize": 'xl',
+      "classes": "ons-u-mb-xl",
+      "body": _("<h1>Your progress has been saved</h1>")
+    })
+  }}
+
+  {% if survey_type and (survey_type in SURVEY_TYPES_BUSINESS or survey_type in SURVEY_TYPES_DEFAULT) %}
```

Comment:
Add nested `if` and check `if survey_type` for all first then check specific types.